examples/deadline/All-In-AWS-Infrastructure-SEP/python/package/lib/storage_tier.py

```diff
@@ -0,0 +1,261 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+from aws_cdk.core import (
+    Construct,
+    Duration,
+    RemovalPolicy,
+    Stack,
+    StackProps
+)
+from aws_cdk.aws_docdb import (
+    BackupProps,
+    DatabaseCluster,
+    InstanceProps,
+    Login
+)
+from aws_cdk.aws_ec2 import (
+    InstanceType,
+    IVpc,
+    SubnetSelection,
+    SubnetType
+)
+from aws_cdk.aws_efs import (
+    FileSystem,
+)
+from aws_cdk.aws_route53 import (
+    IPrivateHostedZone
+)
+
+from aws_rfdk import (
+    MongoDbUsers,
+    MongoDbX509User,
+    DistinguishedName,
+    MongoDbInstance,
+    MongoDbApplicationProps,
+    MongoDbPostInstallSetup,
+    MongoDbSsplLicenseAcceptance,
+    MongoDbVersion,
+    MountableEfs,
+    X509CertificatePem,
+    X509CertificatePkcs12
+)
+
+from aws_rfdk.deadline import (
+    DatabaseConnection
+)
+
+
+@dataclass
+class StorageTierProps(StackProps):
+    """
+    Properties for StorageTier
+    """
+    # The VPC to deploy resources into.
+    vpc: IVpc
+
+
+class StorageTier(Stack):
+    """
+    The storage tier of the render farm.
+    This stack contains all constructs that persist data which would be useful to keep between deployments.
+    There should little to no "business-logic" constructs in this stack.
+    """
+
+    def __init__(self, scope: Construct, stack_id: str, *, props: StorageTierProps, **kwargs):
```

Comment:
Since the storage tier isn't relevant to the SEP we could either:
1. Leave it in, but empty with a comment that directs the customer to the basic example for a demonstration of the storage tier.
2. Remove the storage tier entirely, and leave a comment in the app that says that the storage tier was removed to focus the example and direct the customer to the basic example for the storage tier.

I.e. in either case, just use the self-contained Repository in the service tier.